.pre-commit-config.yaml

```diff
@@ -30,3 +30,37 @@ repos:
     hooks:
     -   id: pyupgrade
         args: [--py37-plus]
+-   repo: https://github.com/pre-commit/pygrep-hooks
+    rev: v1.6.0
+    hooks:
+      - id: rst-backticks
+        exclude: (?x)(
+          text\.rst|
```

Comment:
As follow up would be good to gradually delete these and fix the excluded files

Reply:
Can you add a comment to that effect (uh, assuming yaml supports comments)

Reply:
Added

Reply:
also can you create an issue for this (with checkboxes)